.gitignore

```diff
@@ -9,4 +9,5 @@
 /tmp
 coverage
 node_modules
-package-lock.json
\ No newline at end of file
+oclif.manifest.json
+tsconfig.tsbuildinfo
```

Comment:
Add `.eslintcache`

Reply:
This file is already there. 

https://github.com/vtex/cli-plugin-template/blob/f1b8ccf5cb5f9f677046b9c5b18c3ae1aa89e696/.gitignore#L2

Reply:
Ops, sorry, didn't saw